sklearn/metrics/_regression.py

```diff
@@ -364,7 +364,7 @@ def mean_absolute_percentage_error(
 
     Returns
     -------
-    loss : float or ndarray of floats
+    loss : float or array of floats
```

Comment:
I would rather not update docstrings in this way while array API support is still experimental and not enabled by default in scikit-learn.

We will update the types of arguments and attributes in docstring later.

Reply:
done, thanks!